core/common/src/main/scala/net/liftweb/common/Box.scala

```diff
@@ -814,30 +790,39 @@ sealed abstract class Box[+A] extends Product with Serializable{
   }
 
   /**
-    * Returns a `Full` box containing the results of applying `flipFn` to this box if it is a `Failure`,
-    * `ParamFailure` or `Empty`, and `flipFn` is defined for it. Returns `Empty` if this box is `Full`.
-    * In other words, it "flips" the full/empty status of this Box.
+    * Transforms this box using the `transformFn`. If `transformFn` is defined for this box,
+    * returns the result of applying `transformFn` to it. Otherwise, returns this box unchanged.
```

Comment:
I think we should at a note here that if you are looking to change the value *inside* a `Full` box, you should really be using `map`, and if you're looking to change a `Failure` or `Empty` into a `Full`, you should really be using `flip`.

There's enough complexity in `Box` that providing some helpful guides in some of this method-specific documentation starts making sense for me. What do you think?

Reply:
Yeah, that sounds pretty helpful. Will do so.